salt/metalk8s/addons/logging/fluent-bit/deployed/configmap.sls

```diff
@@ -0,0 +1,104 @@
+include:
+  - ...deployed.namespace
+
+Create fluent-bit ConfigMap:
+  metalk8s_kubernetes.object_present:
+    - manifest:
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: fluent-bit
+          namespace: metalk8s-logging
+          labels:
+            app: fluent-bit
+            app.kubernetes.io/managed-by: salt
+            app.kubernetes.io/name: fluent-bit
+            app.kubernetes.io/part-of: metalk8s
+            heritage: metalk8s
+            release: fluent-bit
+        data:
+          fluent-bit.conf: |-
+            [SERVICE]
+                HTTP_Server    On
+                HTTP_Listen    0.0.0.0
+                HTTP_PORT      2020
+                Flush          1
+                Daemon         Off
+                Log_Level      warn
+                Parsers_File   parsers.conf
+            [INPUT]
+                Name           tail
+                Tag            kube.*
+                Path           /var/log/containers/*.log
+                Parser         docker
+                DB             /run/fluent-bit/flb_kube.db
+                Mem_Buf_Limit  5MB
+            [INPUT]
+                Name              systemd
+                Tag               host.*
+                DB                /run/fluent-bit/flb_journal.db
+                Strip_Underscores On
+            [FILTER]
+                Name           kubernetes
+                Match          kube.*
+                Kube_URL       https://kubernetes.default.svc:443
+                Merge_Log On
+                K8S-Logging.Parser Off
+            [FILTER]
+                Name           modify
+                Match          host.*
+                Rename         HOSTNAME node
```

Comment:
Not sure we should rename it to node because hostname may be different from node name :/
I would instead retrieve the node name of the current host here if it's doable (or add a hostname label on k8s logs but not sure it's easy to do as well)

Reply:
PS look like hostname is not really good, on my test env I got some logs with `localhost` and `localhost.localdomain` as node  (so likely HOSTNAME)
Looking on log it's seems these logs are not really relevant (maybe we should filter out logs with hostname `localhost` and `localhost.localdomain` )

Reply:
It seems to be a little bit tricky to achieve that.
I can't find an easy way to retrieve the real hostname from the kubernetes logs or to retrieve the nodename from journal logs.
Maybe we should just have 2 different labels `host` and `node`.
I've tried to use $HOSTNAME env var, but it's set to the container hostname... not really useful.

Reply:
Ok let's use `hostname` and `pod` for the moment, I updated the dashboard accordingly